src/runtime/providers/cloudinary.ts

```diff
@@ -1,111 +1,126 @@
+//       northWest: 'north_west',
+//       west: 'west',
+//       southWest: 'south_west',
+//       south: 'south',
+//       southEast: 'south_east',
+//       east: 'east',
+//       center: 'center'
+//     }
+//   },
+//   joinWith: ',',
+//   formatter: (key, value) => `${key}_${value}`
+// })
 
 const defaultModifiers = {
   format: 'auto',
   quality: 'auto'
 }
 
 export const getImage: ProviderGetImage = (src, { modifiers = {}, baseURL = '/' } = {}) => {
+  const cloudName = baseURL.replace('https://res.cloudinary.com/', '').split('/')[0]
   const mergeModifiers = defu(modifiers, defaultModifiers)
-  const operations = operationsGenerator(mergeModifiers as any)
+  // const operations = operationsGenerator(mergeModifiers as any)
 
-  const remoteFolderMapping = baseURL.match(/\/image\/upload\/(.*)/)
-  // Handle delivery remote media file URLs
-  // see: https://cloudinary.com/documentation/fetch_remote_images
-  // Note: Non-remote images will pass into this function if the baseURL is not using a sub directory
-  if (remoteFolderMapping?.length >= 1) {
-    // need to do some weird logic to get the remote folder after image/upload after the operations and before the src
-    const remoteFolder = remoteFolderMapping[1]
-    const baseURLWithoutRemoteFolder = baseURL.replace(remoteFolder, '')
+  // const remoteFolderMapping = baseURL.match(/\/image\/upload\/(.*)/)
+  // // Handle delivery remote media file URLs
+  // // see: https://cloudinary.com/documentation/fetch_remote_images
+  // // Note: Non-remote images will pass into this function if the baseURL is not using a sub directory
+  // if (remoteFolderMapping?.length >= 1) {
+  //   // need to do some weird logic to get the remote folder after image/upload after the operations and before the src
+  //   const remoteFolder = remoteFolderMapping[1]
+  //   const baseURLWithoutRemoteFolder = baseURL.replace(remoteFolder, '')
 
-    return {
-      url: joinURL(baseURLWithoutRemoteFolder, operations, remoteFolder, src)
-    }
-  } else if (/\/image\/fetch\/?/.test(baseURL)) {
-    // need to encode the src as a path in case it contains special characters
-    src = encodePath(src)
-  } else {
-    // If the src is not a remote media file then we need to remove the extension (if it exists)
-    src = removePathExtension(src)
-  }
+  //   return {
+  //     url: joinURL(baseURLWithoutRemoteFolder, operations, remoteFolder, src)
+  //   }
+  // } else if (/\/image\/fetch\/?/.test(baseURL)) {
+  //   if (/\/image\/fetch\/?/.test(baseURL)) {
+  //   // need to encode the src as a path in case it contains special characters
+  //   src = encodePath(src)
+  // } else {
+  //   // If the src is not a remote media file then we need to remove the extension (if it exists)
+  //   src = removePathExtension(src)
+  // }
 
   return {
-    url: joinURL(baseURL, operations, src)
+    url: constructCloudinaryUrl({
+      options: {
+        src: src.substring(1),
+        ...mergeModifiers
```

Comment:
are the width and height getting passed in here? i'm not following if there's a place that they get added to the modifiers as it would, which i wouldn't expect given they're modifiers, but if not it might make sense to pass the width/height through to the options individually?

Reply:
Yes, they were merged using defu (think of it as merge two object effectively). Now it is not needed as :)